app/views/studies/_new.html.erb

```diff
@@ -1,15 +0,0 @@
-<script type="text/javascript">
-  var CREATE_INVESTIGATION_PATH='<%= create_investigation_path %>';
-</script>
-<% investigations = Investigation.all.select &:can_view? -%>
-<% if investigations.blank? %>
-  <%= button_link_to("New #{t('investigation')}", 'arrow_right', new_investigation_path) -%>
-<% else %>
-  <h1>New <%= t('study') %></h1>
-
-  <div class="asset_form">
-    <%= form_for @study do |f| %>
-      <%= render :partial => "studies/form", :locals => { :f => f, :action=>:new} -%>
-    <% end -%>
-  </div>
-<% end -%>
```

Comment:
I'm unclear what this has been replaced with, and doesn't it just leave the file blank?
It will lose the prompt to provide an Investigation if none are already available.